resources/test/dumps/Sandpile.js

```diff
@@ -245,9 +245,11 @@ function dropPatch() {
     return ListPrims.oneOf(world.patches());
   }
   if ((Prims.equality(world.observer.getGlobal('drop-location'), "mouse-click") && MousePrims.isDown())) {
-    Prims.every(0.3, function() {
+    var dropPatch_0;
+    if (Prims.everyTimerElapsed(0.3, dropPatch_0)) {
       return world.getPatchAt(MousePrims.getX(), MousePrims.getY());
-    }, 'dropPatch-0');
+      dropPatch_0 = new Prims.everyTimer();
+    }
```

Comment:
This implementation of `every` is incorrect—twice!  The code for setting the `every` timer here is occurring on a line that will never be executed (because of the `return` on the preceding line).  Even if this line were moved up, though, this would still be incorrect, because `dropPatch_0` will not persist past the end of the block in which the `everyTimerElapsed` occurs.  The way that this primitive is implemented, I don't see any way for the timer argument to `everyTimerElapsed` to ever be anything but `undefined`, even if `dropPatch_0 = new Prims.everyTimer();` *were* executed.

Simply put, `every` cannot work if its timer is scoped to the same block as the `everyTimerElapsed` call.